src/main/features/response/views/claim-details.njk

```diff
@@ -16,7 +16,8 @@
       {{ amountBreakdownTable('View amount breakdown', claim, interestData) }}
       <p><span class="bold-small">{{ t('Reason for claim') }}:</span></p>
       <p>{{ claim.claimData.reason }}</p>
-
+      <p><span class="bold-small">{{ t('Download the claim') }}</span></p>
+      <p><span class="block"><a href="{{ DashboardPaths.claimReceiptReceiver.evaluateUri({ externalId: claim.externalId }) }}">{{ t('Download claim') }}</a>{{ t(' (PDF)') }}</span></p>
```

Comment:
The same here, use `ClaimPaths.receiptReceiver` directly. I started removing aliases such as `DashboardPaths.claimReceiptReceiver` from path files as they are just confusing.

Reply:
Also please remove whitespace from `' (PDF)'`, we do not want to have to add it into translation files.